tools/minikube/instant-demo.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+set -u
+
+: "${USE_SSL:=false}"
+: "${backend_proto:="http"}"
+: "${S3GW_CHARTS:="$(dirname "$0")/../../charts/charts/s3gw"}"
+
+if [[ ! -r "${S3GW_CHARTS}/Chart.yaml" ]]; then
+    echo "S3GW_CHARTS set to something weird ($S3GW_CHARTS)"
+    echo "Please set to the s3gw charts dir (e.g s3gw-charts/charts/s3gw)"
+    exit 1
+fi
```

Comment:
Instead of bailing when no local charts repo is present, why not do
`helm repo add s3gw https://aquarist-labs.github.io/s3gw-charts` and install an official release?

Reply:
Good idea! It's an even better default